docs/how-to-guides/block-tutorial/writing-your-first-block-type.md

```diff
@@ -1,137 +1,180 @@
 
-While the block's editor behaviors are implemented in JavaScript, you'll need to register your block server-side to ensure that the script is enqueued when the editor loads. Register scripts and styles using [`wp_register_script`](https://developer.wordpress.org/reference/functions/wp_register_script/) and [`wp_register_style`](https://developer.wordpress.org/reference/functions/wp_register_style/), then assign these as handles associated with your block using the `script`, `style`, `editor_script`, and `editor_style` block type registration settings.
+This guide focuses on just the block, see the [Create a Block tutorial](/docs/getting-started/create-block/README.md) for a complete setup.
 
-The `editor_script` and `editor_style` files will only be enqueued in the editor, while the `script` and `style` will be enqueued both in the editor and when viewing a post on the front of your site.
+## Before you start
+
+Static blocks are implemented in JavaScript, so a basic level of JavaScript is helpful, see the [Getting Started with JavaScript](/docs/how-to-guides/javascript/README.md) for a refresher.
+
+Blocks are added to WordPress using plugins, so you will need:
+
+-   WordPress development environment, see [setup guide](/docs/getting-started/devenv/README.md)
+-   JavaScript build tools (node/npm) if using JSX example
+
+## Step-by-step guide
+
+### Step 1: Configure block.json
+
+The functions of a static block is defined in JavaScript, however the settings and other metadata should be defined in a block.json file.
+
+Here are the basic settings:
+
+-   `title`: Block title shown in inserter
+-   `name`: Unique name defines your block
+-   `category`: Category in inserter (text, media, design, widgets, theme embed)
+-   `icon`: Dashicon icon displayed for block
+-   `editorScript`: JavaScript file to load for block
+
+The block.json file should be added to your plugin. To start a new plugin, create a directory in `/wp-content/plugins/` in your WordPress.
+
+Create a basic block.json file there:
+
+{% codetabs %}
+{% JSX %}
+
+```json
+{
+	"apiVersion": 2,
+	"name": "gutenberg-examples/example-01-basic-esnext",
+	"title": "Example: Basic (ESNext)",
+	"icon": "universal-access-alt",
+	"category": "layout",
+	"example": {},
+	"editorScript": "file:./build/index.js"
+}
+```
+
+{% Plain %}
+
+```json
+{
+	"apiVersion": 2,
+	"title": "Example: Basic",
+	"name": "gutenberg-examples/example-01-basic",
+	"category": "layout",
+	"icon": "universal-access-alt",
+	"editorScript": "file:./block.js"
+}
+```
+
+{% end %}
+
+### Step 2: Register block in plugin
+
+With the block.json in place, the registration for the block is a single function call in PHP, this will setup the block and JavaScript file specified in the `editorScript` property to load in the editor.
```

Comment:
Code highlight for conistency:
```suggestion
With the `block.json` in place, the registration for the block is a single function call in PHP, this will setup the block and JavaScript file specified in the `editorScript` property to load in the editor.
```